packages/common/src/core/types/IHubItemEntity.ts

```diff
@@ -41,6 +41,11 @@ export interface IHubItemEntity extends IHubEntityBase, IWithPermissions {
    */
   extent?: number[][];
 
+  /**
+   * If the item has discussions enabled
+   */
+  isDiscussable?: boolean;
```

Comment:
The notion of something being discussable currently applies to both groups and content. Maybe this would be better defined in an `IWithDiscussions` interface in the `traits` directory and this interface extend `IWithDiscussions`. Make sure you import/export this new trait interface from the `index.ts` file in the `traits` directory.